src/Config/Compute.hs

```diff
@@ -72,7 +72,7 @@ findExp name vs bod = [SettingMatchExp $
         rhs = apps' $ map noLoc $ HsVar noExtField (noLoc name) : map snd rep
 
         rep = zip vs $ map (mkVar . pure) ['a'..]
-        f (HsVar _ x) | Just y <- lookup (occNameString $ occName $ unLoc x) rep = y
+        f (HsVar _ x) | Just y <- lookup (rdrNameStr x) rep = y
```

Comment:
Could we get some hlint rules for these in the HLint local hints thing? Otherwise I'll reintroduce them :)